digitaltwins/device/src/digital_twin_client.ts

```diff
@@ -554,192 +455,76 @@ export class DigitalTwinClient {
     }
 
     return callbackToPromise((_callback) => {
-      if (!this._interfaceInstances[interfaceInstanceName].registered) {
-        return _callback(new Error(interfaceInstanceName + ' is not registered'));
-      } else {
-        /* Codes_SRS_NODE_DIGITAL_TWIN_DEVICE_06_038: [** Properties may invoke the method `report` with a value to produce a patch to the reported properties. **] */
-        let interfaceInstancePart = interfaceInstancePrefix + interfaceInstanceName;
-        let propertyContent: any = {
-          value: propertyValue
+      let iName = interfaceInstance.interfaceInstanceName;
```

Comment:
>let iName = interfaceInstance.interfaceInstanceName; [](start = 6, length = 52)

Only use this variable once?  No need for it then.  Also the indentation is wrong after this.